app/views/lobby/home.scala

```diff
@@ -29,7 +29,7 @@ object home:
               Json.obj("minutes" -> pb.mins, "remainingSeconds" -> (pb.remainingSeconds + 3))
           )
       ),
-      moreCss = cssTag("lobby"),
+      moreCss = frag(cssTag("lobby"), cssTag("dailyFeed")),
```

Comment:
the necessary css to render lobby daily feed should be within the lobby CSS file. With a distinct daily feed CSS for edit/update.